src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -152,6 +153,12 @@ public DesiredCapabilities buildCapabilities(ConductorConfig config) {
             capabilities.setCapability(MobileCapabilityType.AUTOMATION_NAME, config.getAutomationName());
         }
 
+        if (config.getCustomCapabilities().size() > 0) {
```

Comment:
Do you actually have to check this before itterating? Because if so that's crazy.

Reply:
You are right, I don't have to.